opencv/c_extension/fast_image_processing.cpp

```diff
@@ -0,0 +1,229 @@
+#include <string>
+#include <iostream>
+#include <vector>
+#include <math.h> // atan
+#include <cmath> // std::abs
+
+#include <pybind11/pybind11.h>
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/core/eigen.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+
+#define CONVERT_TO_INT true
+#define MAX_WIDTH 320
```

Comment:
not be a define, image size may change at running time. At least with out compilation

Reply:
True, we also need to define the ROI (REGIONS) dynamically... if you have any good idea to do it in a clean way, please share it ;)